psl-core/src/main/java/org/linqs/psl/model/rule/arithmetic/AbstractArithmeticRule.java

```diff
@@ -108,6 +110,16 @@ public ArithmeticRuleExpression getExpression() {
 		return expression;
 	}
 
+	@Override
```

Comment:
Instead of having this in all the sub-classes, just make Rule an abstract class.